packages/workspace/src/utils/plugins/installed-plugins.ts

```diff
@@ -1,50 +1,36 @@
 import { terminal } from '@angular-devkit/core';
-import { readdirSync } from 'fs';
-import * as path from 'path';
+import { readJsonFile } from '../fileutils';
 import { output } from '../output';
 import { CommunityPlugin, CorePlugin, PluginCapabilities } from './models';
 import { getPluginCapabilities } from './plugin-capabilities';
 import { hasElements } from './shared';
 
-function getPackagesFromNodeModules(
-  workspaceRoot: string,
-  requery: boolean = false
-): string[] {
-  let packageList: string[] = [];
-
-  if (!requery && packageList.length > 0) {
-    return packageList;
-  }
-
-  const nodeModulesDir = path.join(workspaceRoot, 'node_modules');
-  readdirSync(nodeModulesDir).forEach((npmPackageOrScope) => {
-    if (npmPackageOrScope.startsWith('@')) {
-      readdirSync(path.join(nodeModulesDir, npmPackageOrScope)).forEach((p) => {
-        packageList.push(`${npmPackageOrScope}/${p}`);
-      });
-    } else {
-      packageList.push(npmPackageOrScope);
-    }
-  });
-
-  return packageList;
-}
-
 export function getInstalledPluginsFromNodeModules(
```

Comment:
Might want to change the name of this function now?

Reply:
As proposed changes to `getInstalledPluginsFromPackageJson`.